RefApp/ViewControllers/EnvironmentViewController.swift

```diff
@@ -0,0 +1,153 @@
+        topHolderView.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 1/3).isActive = true
+        
+        logoImageView.anchor(top: nil, bottom: topHolderView.bottomAnchor, leading: nil, trailing: nil, padding: .init(top: 0, left: 0, bottom: 0, right: 0), size: .init(width: 0, height: 0))
+        logoImageView.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        
+        bottomHolderView.anchor(top: topHolderView.bottomAnchor, bottom: nil, leading: view.safeAreaLayoutGuide.leadingAnchor, trailing: view.safeAreaLayoutGuide.trailingAnchor)
+        bottomHolderView.heightAnchor.constraint(equalTo: view.heightAnchor, multiplier: 2/3).isActive = true
+        
+        environmentUrlTextField.anchor(top: bottomHolderView.topAnchor, bottom: nil, leading: view.safeAreaLayoutGuide.leadingAnchor, trailing: view.safeAreaLayoutGuide.trailingAnchor, padding: .init(top: 20, left: 16, bottom: 0, right: -16))
+        
+        customerNameTextField.anchor(top: environmentUrlTextField.bottomAnchor, bottom: nil, leading: view.safeAreaLayoutGuide.leadingAnchor, trailing: view.safeAreaLayoutGuide.trailingAnchor, padding: .init(top:20, left: 16, bottom:0, right: -16))
+        
+        businessUnitTextField.anchor(top: customerNameTextField.bottomAnchor, bottom: nil, leading: view.safeAreaLayoutGuide.leadingAnchor, trailing: view.safeAreaLayoutGuide.trailingAnchor, padding: .init(top:20, left: 16, bottom:0, right: -16))
+        
+        nextButton.anchor(top: businessUnitTextField.bottomAnchor, bottom: nil, leading: view.safeAreaLayoutGuide.leadingAnchor, trailing: view.safeAreaLayoutGuide.trailingAnchor, padding: .init(top:20, left: 16, bottom:0, right: -16))
+    }
+}
+
+
+// MARK: - Navigation
+extension EnvironmentViewController {
+    
+    /// Navigate Login Page after validation
+    @objc fileprivate func navigateToLogin() {
+        
+        view.endEditing(true)
+        
+        guard let environmentUrl = environmentUrlTextField.text,
+            let customerName = customerNameTextField.text,
+            let businessUnit = businessUnitTextField.text,
+            environmentUrl != "", customerName != "", businessUnit != "" else {
+                
+                let okAction = UIAlertAction(title: "Ok", style: .cancel, handler: {
+                    (alert: UIAlertAction!) -> Void in
+                })
+                
+                self.popupAlert(title:NSLocalizedString("Sorry", comment: "") , message: NSLocalizedString("Please fill all fields", comment: ""), actions: [okAction], preferedStyle: .alert)
+                return
+        }
+        
+        let environment = Environment(baseUrl: environmentUrl, customer: customerName, businessUnit: businessUnit)
+        
+        let loginViewController = LoginViewController()
```

Comment:
Remove or comment away until implementation is present

Reply:
noted